mcs/class/corlib/System/TimeZoneInfo.WinRT.cs

```diff
@@ -338,8 +338,17 @@ internal static List<TimeZoneInfo> GetSystemTimeZonesWinRTFallback ()
 				// EnumDynamicTimeZoneInformation() might not be available.
 			}
 
-			if (result.Count == 0)
+			if (result.Count == 0) {
 				result.Add (Local);
+				result.Add (Utc);
```

Comment:
Should one or both of these always be added? Or is it best to have fewer/one and leave it alone if not zero?

Reply:
Yes in case there are not time zones on the system (cases where EnumDynamicTimeZoneInformation is not available) we should add them to the result. This logic was already in place but it didn't add UTC, so  there are test failing due to not having a UTC time zone returned from GetSystemTimeZones.

Reply:
My question is -- should the `if` be removed or modified?
Should Local and Utc always be present?
Or should Utc always be present?
Or does the dynamic case always add them, so if not zero, they are present anyway?

Reply:
Yes, the dynamic case should already add them to the list (all test pass on system that supports that scenario), so this only handle the special case where no time zones have been added.